### PR TITLE
Capstone project

### PR DESCRIPTION
**Pull request by nimplay**
This is my capstone project using Javascript CSS and HTML

explanatory video [https://www.loom.com/share/2d165aa6566c42bebbd7d7678b4217a0](url)
excuse my stage fright XD

### CODE REVIEW

**Comment by nimplay** [comment]
Summary:
This is the version of the capstone web conference pages emulating a viseo games jam I hope you like it

File: index.html [hunk 1]
```diff
@@ -0,0 +1,125 @@
...
+       <img src="./images/lumen.png" alt="">
+       <img src="./images/line-friends.png" alt="">   
+     </div>
+   </div>  
+   </section>
+   <footer class="footer">
+    <img class="footer-img" src="./images/cck-logo1.png" alt="">
+    <p class="text-gray">The logo of CC Global Summit 2015 was  
+      Lorem ipsum dolor sit, amet consectetur adipisicing elit. Temporibus. </p>
+  </footer>
```
Comment (vechicin) [inline index.html]: I understand you were following the template design, however, according to the project requirements, the footer needs to be present in both versions of both pages. Please add a footer to the mobile version's home page.

Screen Shot 2022-06-23 at 8 53 43

File: about.html [hunk 2]
```diff
@@ -0,0 +1,129 @@
...
+            <a href="./index.html" class="text-orange top-text-var">Home</a>
+            <a href="./about.html" class="text-gray top-text-var">About</a>
+            <a id="button-main-bar" href="https://nimplay.itch.io/">Jam</a>
+          </div>
+        </nav>
+      </header>
+      <section class="home" id="hero">
+        <div class="home-titles">
+          <h2 id="hello" class="text-orange">"Hello world"</h2>
+          <h1 id="maintitle" class="text-orange">NIMPLAY jam 2023</h1>
```
Comment (vechicin) [inline about.html:57]: According to the project requirements and the header hierarchization and accessibility standards, there should be one and only one `<h1>` per page. You have several of them. Please fix this.

Screen Shot 2022-06-23 at 9 04 12

File: about.html [hunk 3]
```diff
@@ -0,0 +1,129 @@
...
+      <h2 id="program-title">
+        Nimplay<br />
+        2017
+      </h2>
+      <div class="red-line"></div>
+      <p class="home-date text-gray">
+        The logo of NIMPLAY video game development.
+      </p>
+      <div id="program-card">
+        <img src="./images/nimplayLogo.jpg" alt="" />
```
Comment (vechicin) [inline about.html]: Please add an alternative text to all of your images.

Screen Shot 2022-06-23 at 9 05 25

File: about.html [hunk 4]
```diff
@@ -0,0 +1,129 @@
...
+          <img src="./images/mozilla.png" alt="" />
+        </div>
+        <div class="sponsor2">
+          <img src="./images/lumen.png" alt="" />
+          <img src="./images/line-friends.png" alt="" />
+        </div>
+      </div>
+    </section>
+    <footer class="footer">
+      <img src="./images/cck-logo1.png" alt="" />
```
Comment (vechicin) [inline about.html]: Please add alternative text to all of your images.

Screen Shot 2022-06-23 at 9 07 00

File: index.html [hunk 5]
```diff
@@ -0,0 +1,125 @@
...
+       <img src="./images/mozilla.png" alt="">  
+     </div>
+     <div class="sponsor2"> 
+       <img src="./images/lumen.png" alt="">
+       <img src="./images/line-friends.png" alt="">   
+     </div>
+   </div>  
+   </section>
+   <footer class="footer">
+    <img class="footer-img" src="./images/cck-logo1.png" alt="">
```
Comment (vechicin) [inline index.html]: Please add alternative text to all of your images
Screen Shot 2022-06-23 at 9 08 45

File: index.html [hunk 6]
```diff
@@ -0,0 +1,125 @@
...
+    </div>
+    <div class="speaker-data">
+      <img class="speaker-image" src="./images/speaker_02.png" alt="">
+      <div class="speaker-data2">
+      <h2 class="speaker-title">Nahomi Noh</h2>
+      <p class="speaker-paragraph">Professor in the Video Game University .</p>
+        <div class="speaker-line"></div>
+      <p class="speaker-paragraph2">As the main venue for new media art production in 
+        Korea. Lorem ipsum dolor sit amet consectetur.</p>
+    </div>
```
Comment (vechicin) [inline index.html]: According to the project requirements, the content of this section should be dynamically generated in its entirety.

Screen Shot 2022-06-23 at 9 15 38

**Review by vechicin — CHANGES_REQUESTED** [review]
# Hi @nimplay ✋, 

Good job so far! 🔥 
There are some issues that you still need to work on to go to the next project but you are almost there! 🚨 
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Highlights ⚡ 

- [x] Good video presentation
- [x] Project works as intended 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions 📂 
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 6 above)
Comment (nimplay) [reply index.html]: sorry but the template have two speaker static and the rest appear clicking on the buttom. but all the main program is generated dynamically.  I think I did even more than what was required in the assignment in this section.

File: about.html (hunk 2 above)
Comment (nimplay) [reply about.html:57]: ok

File: index.html (hunk 5 above)
Comment (nimplay) [reply index.html]: ok I copy!

**Comment by nimplay** [comment]
Sumary:
All alt atribute was added and the all speaker card are display dinamically.

File: about.html [hunk 7]
```diff
@@ -108,18 +108,18 @@ <h1>2021</h1>
         <h2 id="program-title" class="text-color">Sponsors</h2>
         <div id="red-line2"></div>
         <div class="sponsor">
-          <img src="./images/arcadia.png" alt="" />
-          <img src="./images/kcc.png" alt="" />
-          <img src="./images/mozilla.png" alt="" />
+          <img src="./images/arcadia.png" alt="Arcadia" />
+          <img src="./images/kcc.png" alt="KCC" />
+          <img src="./images/mozilla.png" alt="Mozilla" />
```
Comment (mateo951) [inline about.html:113]: Addressed ✔

File: app.js [hunk 8]
```diff
@@ -1,51 +1,77 @@
+const displayCard = [
```
Comment (mateo951) [inline app.js:1]: Addressed 🚀

**Review by mateo951 — APPROVED** [review]
### Hi @nimplay, 

Well done implementing all the changes required by the previous reviewer 💪🤓
In my opinion, your project is ready for the presentation!  There is nothing else to say other than... it's time to merge it 🔥 

### Good luck! 🎉 

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**Pull request merged**